modules/auxiliary/gather/ldap_esc_vulnerable_cert_finder.rb

```diff
@@ -425,6 +434,25 @@ def print_vulnerable_cert_info
       vulns.delete('ESC3_TEMPLATE_2') unless any_esc3t1 # don't report ESC3_TEMPLATE_2 if there are no instances of ESC3_TEMPLATE_1
       next if vulns.empty?
 
+      vulns.each do |vuln|
+        vuln = 'ESC3' if vuln == 'ESC3_TEMPLATE_1'
+        next if vuln == 'ESC3_TEMPLATE_2'
```

Comment:
Isn't this condition already checked at line 434 with `vulns.delete('ESC3_TEMPLATE_2') unless any_esc3t1 # don't report ESC3_TEMPLATE_2 if there are no instances of ESC3_TEMPLATE_1`?

Reply:
No `ESC3_TEMPLATE_2` will still be present here if there are any `ESC3_TEMPLATE_1` templates identified. ESC3_TEMPLATE_2 isn't really a vulnerable template so we always skip reporting it as a vulnerability here, but the outer loop prints it for the user because it's useful information when running the full attack workflow for ESC3.